gcloud-java-examples/README.md

```diff
@@ -89,6 +89,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
```

Comment:
I see what you are saying: from their name it's easy to tell what the other examples are meant to demonstrate. So I added "illustrating the use of the gcloud-java-nio jar" here to clarify. I chose this over your suggestion because to just use NIO, the usual case is not the jar but it's to have nio as a dependency.

Since people sometimes use the examples as guidance about what to do, perhaps we should have an example that does that too? The problem there is that if gcloud-java-examples has gcloud-java-nio as a dependency already then the Stat example is less convincing.

On the other hand it will allow us to have consistent instructions in the gcloud-java-examples project ("use -Dexe.mainClass"), and then a separate gcloud-java-example2 (?) jar with the different instructions ("use mvn package -Dlotsofoptions").

What do you guys think?